packages/manager/src/components/MaintenanceBanner/LinodeMaintenanceBanner.tsx

```diff
@@ -0,0 +1,62 @@
+import { useAllAccountMaintenanceQuery } from '@linode/queries';
+import { Notice, Typography } from '@linode/ui';
+import React from 'react';
+
+import { PENDING_MAINTENANCE_FILTER } from 'src/features/Account/Maintenance/utilities';
+import { isPlatformMaintenance } from 'src/hooks/usePlatformMaintenance';
+
+import { DateTimeDisplay } from '../DateTimeDisplay';
+import { Link } from '../Link';
+
+import type { Linode } from '@linode/api-v4';
+
+interface Props {
+  linodeId: Linode['id'] | undefined;
```

Comment:
Can we just simplify `Linode['id']` to `number`?
```suggestion
  linodeId: number | undefined;
```

Reply:
Actually nvm, the field name is enough I suppose.